TODO.md

```diff
@@ -0,0 +1,77 @@
+# Add support for yml typed manifests
+
+- [ ] Blocked by <https://github.com/cloudbase/powershell-yaml/issues/38>
+- [x] .vscode settings
+- [x] Submodule
+- [x] `parse_json` -> `Scoop-ParseManifest { if (json) { _parseJson } else {_parseYaml} }`
+- [x] url_manifest function
+- [ ] ConvertTo-Yaml is converting `|` to `>` (See: <https://github.com/cloudbase/powershell-yaml/issues/40>)
+- [ ] Binaries
+    - [x] autoupdate function call
+    - [ ] auto-pr
+        - [ ] Inspect TODOs
+    - [x] checkurls
+    - [x] checkver
+        - [x] Fix noteproperty
+    - [x] describe
+    - [x] formatjson
+    - [x] format
+    - [x] missing-checkver
+    - [x] test
+- [ ] Commands
+    - [x] install.ps1
+- [ ] Remote install yaml
+- [ ] Tests
+    - [ ] Validate yaml
+- [ ] `bin\*`
+    - [ ] Import Yaml module
+- [ ] `Scoop-Format-Manifest.Tests.ps1` -> `Scoop-Format-Manifest.json.Tests.ps1`
+- [ ] `Scoop-Format-Manifest.yml.Tests.ps1`
+- [ ] `ConvertToPrettyYaml` ❓❓
+    - [ ] Indent arrays with 2 spaces
+    - [ ] Convert `>` to `|` (workarround)
+
+# Local testing
+
+Predefined docker for adequate testing.
+
+1. [ ] Make sure implementation work same for json and yaml.
+1. [ ] Commands
+    1. [ ] Installing
+        1. [ ] Bucket folder
+            - `scoop install yamBoth` and `scoop install yamApache` should produce same output
```

Comment:
I guess one should make automated testing tool which would test every manifest. And an automatic converter, of course. :)

Reply:
converter would look only like:

```powershell
param ($App, $Dir)
. "$env:SCOOP\supporting\yaml\powershell-yaml.psm1"

Get-ChildItem $DIR "$App.json" | % {
    Scoop-WriteManifest "$_.yaml" (Scoop-ParseManifest $_)
}
```

But it' last thing to do. There is still so much hardcoded extensions to refactor and someone would prefer json over yaml in his bucket.